src/globalStyles/dataTableStyles.js

```diff
@@ -23,56 +21,201 @@ export const dataTableColors = {
 };
 
 export const dataTableStyles = {
-  text: {
+  cellContainer: {
```

Comment:
if this PR is just removing old code and moving stuff around.. why the new styles?

Reply:
sorry maybe I shouldv'e been more clear. I created a new file `newDataTableStyles` and made new styles in there for the new data table to ensure everything was still working throughout the refactor. I then shifted the code from `newDataTableStyles` to `dataTableStyles` and removed `newDataTableStyles` - maybe this was a poor decision

Reply:
no - it's all good. just checking that nothing has snuck in that might cause a conflict later on